draft-linus-trans-gossip-ct.md

```diff
@@ -179,20 +181,20 @@ corresponding certificate chain for later use in feedback.
 
 When the client later reconnects to any HTTPS server for the same
 domain it again receives a set of SCTs. The client MUST update its
-store of SCTs for the domain and MUST send to the server the ones in
-the store that were not received from that server.
+store of SCTs [should also discard SCTs from unknown logs] for the domain and MUST send to the server the ones in
+the store that were not received from that server [that are for that server].
```

Comment:
On Mon 2015-07-06 16:31:20 -0400, Linus Nordberg wrote:
>> @@ -179,20 +181,20 @@ corresponding certificate chain for later use in feedback.
>>  
>>  When the client later reconnects to any HTTPS server for the same
>>  domain it again receives a set of SCTs. The client MUST update its
>> -store of SCTs for the domain and MUST send to the server the ones in
>> -the store that were not received from that server.
>> +store of SCTs [should also discard SCTs from unknown logs] for the domain and MUST send to the server the ones in
>> +the store that were not received from that server [that are for that server].
>
> Does this capture it?
>
>> When the client later reconnects to any HTTPS server for the same
> domain it again receives a set of SCTs. The client MUST add new SCTs
> from known logs to its store of SCTs for the server. The client MUST
> send to the server the ones in the store that are for that server and
> were not received from that server.

I'm not convinced this captures it.  if i connect to server X at time T0
and get SCT+certchain S0, and then later i connect at time T1 and get
SCT+certchain S1, i got both S0 and S1 from X.  The text above seems to
imply that i shouldn't send either of them to X.

      --dkg